package.json

```diff
@@ -9,7 +9,7 @@
 		"test:cross-platform:build": "pnpm run --dir packages/kit test:cross-platform:build",
 		"test:vite-ecosystem-ci": "pnpm test --dir packages/kit",
 		"check": "pnpm -r check",
-		"lint": "pnpm -r lint",
+		"lint": "pnpm -r lint && eslint packages/**/*.js",
```

Comment:
What happens on shells that don't implement `**`? Does ESLint also implement this internally, and do we want to enclose this in quotes so the shell doesn't try to handle it?

Reply:
```suggestion
		"lint": "pnpm -r lint && eslint 'packages/**/*.js'",
```

Reply:
hmm. putting it in quotes makes the CI fail, so that's probably not the way to go. It was without quotes previously and no one complained, so hopefully we can leave it like that for now

Reply:
Are you sure it's not failing because there are additional files being linted in CI that are getting missed when you are relying on the shell to handle the globstars?

Reply:
yeah, maybe you were right. I got it working now